extension/src/controllers/subtitle-controller.ts

```diff
@@ -154,6 +169,22 @@ export default class SubtitleController {
             this.cacheHtml();
         }
 
+        const newAlignments = allTextSubtitleSettings(newSubtitleSettings).map((s) => s.subtitleAlignment);
+        if (newAlignments !== this.subtitleTrackAlignments) {
```

Comment:
This check will always return `true` because JavaScript will use object identity for comparison. This class already implements `_arrayEquals`, you can use that instead.